src/metabase/api/dashboard.clj

```diff
@@ -49,6 +49,34 @@
 
 (set! *warn-on-reflection* true)
 
+(defn- dashboards-list [filter-option]
+  (as-> (t2/select :model/Dashboard {:where    [:and (case (or (keyword filter-option) :all)
+                                                      (:all :archived)  true
+                                                      :mine [:= :creator_id api/*current-user-id*])
+                                                [:= :archived (= (keyword filter-option) :archived)]]
+                                     :order-by [:%lower.name]}) <>
+    (t2/hydrate <> :creator)
+    (filter mi/can-read? <>)))
+
+(api/defendpoint GET "/"
```

Comment:
Do we have any way to mark deprecation? E.g. `^:deprecated`. Just wondering. No action if the answer is no.

Reply:
Good point, I think we've used `^:deprecated` with functions before so I can do the same thing here but add it to the endpoint method. Maybe we can use this data at some point. I've added it 953c0b8b3b738355688e010b38e75506cfc4e36e